### DISCUSSION TITLE
Feature Request: update the widgets only once if `update_interval` is negative

### DISCUSSION THREAD

**Discussion by mrtnvgr** [qtile/qtile#4083] [Ideas] [answered]
As the title says. For example, I would want to get the weather information only when qtile starts.

Currently `update_interval < 0` behaves like `update_interval = 0`.

**Comment by m-col** [comment]
I like the concept and the use-case makes sense. However, maybe the implementation would be more clear if it used `update_interval=None`. +1 from me though

**Comment by mrtnvgr** [comment]
I will make a PR for this when I have free time.

**Comment by mrtnvgr** [comment, marked as answer]
#4085
